src/event/mod.rs

```diff
@@ -55,7 +55,16 @@ impl EventHandler {
     pub fn schedule_with<F>(&self, arg_cb: F)
         where F: FnOnce(&mut EventContext, Handle<JsValue>, Handle<JsFunction>),
               F: Send + 'static {
-        let callback = Box::into_raw(Box::new(arg_cb)) as *mut c_void;
+        // HACK: Work around for race condition in `close`. `EventHandler` cannot be
+        // dropped until all callbacks have executed.
+        // NOTE: This will still leak memory if the callback is never called
+        let cloned_cb = self.clone();
+        let cb = move |cx, this, cb| {
+            arg_cb(cx, this, cb);
+            std::mem::drop(cloned_cb);
```

Comment:
@dherman When you are holding onto a variable just to control when it's dropped, is it more idiomatic to manually call `std::mem::drop` or to assign to an un-used variable? `let _cloned_cb = cloned_cb;`

Reply:
I see `let _ = cloned_cb` used maybe a bit more than `drop(cloned_cb)` but both are common AFAIK